### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by brad**
@orcasgit/orcas-developers Please review. Updated the travis/tox test matrix and supported versions of django, and added docstrings for all models, help_text for all fields.

### CODE REVIEW

File: test_settings.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    'loggers': {
+        'withingsapp.tasks': {'handlers': ['null'], 'level': 'DEBUG'},
+    },
+}
+
+MIDDLEWARE_CLASSES = (
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+)
```
Comment (grokcode) [inline test_settings.py]: @brad 👍  I like this style for handling the settings. Much cleaner.

File: withingsapp/migrations/0002_help_text.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2016-11-03 18:33
```
Comment (grokcode) [inline withingsapp/migrations/0002_help_text.py]: @brad To be safe, I think we should generate the migrations with the minimum supported Django version (1.8). Higher versions of Django should be backwards compatible, but lower versions aren't always able to handle the migrations generated by the higher versions. https://docs.djangoproject.com/en/1.10/topics/migrations/#supporting-multiple-django-versions

File: test_settings.py (hunk 1 above)
Comment (brad) [inline test_settings.py]: :neckbeard:

**Comment by brad** [comment]
@grokcode I think this is all better, ready for :eyes: again

**Comment by grokcode** [comment]
LGTM! :shipit:

**Pull request merged**